pkg/controller/csr_check.go

```diff
@@ -44,7 +42,8 @@ var nodeBootstrapperGroups = sets.NewString(
 	"system:authenticated",
 )
 
-var maxPendingCSRs uint32
+var MaxPendingCSRs uint32
+var PendingCSRs uint32
```

Comment:
Do these need to be public?

Reply:
Yes, those values are exposed for metrics.